### PR TITLE
Update to RC2

### PR DESCRIPTION
**Pull request by nblumhardt**

### CODE REVIEW

**Comment by nblumhardt** [comment]
cc @merbla

File: src/Serilog.Enrichers.Thread/project.json [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
-  "compilationOptions": {
-    "keyFile": "../../assets/Serilog.snk"
+  "buildOptions": {
+    "keyFile": "../../assets/Serilog.snk",
+    "xmlDoc": true,
+    "warningsAsErrors": true
   },
   "frameworks": {
-    "net45": {
+    "net4.5": {
```
Comment (khellang) [inline src/Serilog.Enrichers.Thread/project.json:20]: Is this correct?

Comment (nblumhardt) [inline src/Serilog.Enrichers.Thread/project.json:20]: I believe so - all of the new TFMs seem to use this pattern. Works for https://github.com/serilog/serilog/blob/dev/src/Serilog/project.json#L16, anyway :-)

Comment (khellang) [inline src/Serilog.Enrichers.Thread/project.json:20]: Huh. That's really weird. Why would they change existing TFMs? Not even MS themselves use that style; https://github.com/aspnet/KestrelHttpServer/blob/dev/src/Microsoft.AspNetCore.Server.Kestrel/project.json#L24

Comment (nblumhardt) [inline src/Serilog.Enrichers.Thread/project.json:20]: Falls out of the table here: https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#mapping-the-net-platform-standard-to-platforms

Comment (khellang) [inline src/Serilog.Enrichers.Thread/project.json:20]: And right below, where it lists the NuGet identifiers, it's lists the good old; .NET Framework 2.0 - 4.6 -> net20 - net46 😝 Anyway... As long as it works.

Comment (nblumhardt) [inline src/Serilog.Enrichers.Thread/project.json:20]: Hehe :-) .. yeah! I'm just desperate for a bit of sanity/consistency, so happy if they support the dotted names throughout :+1: 

:shipit: ?

**Comment by nblumhardt** [comment]
(Assuming a "yes" ;-) ) .. need to get this out to unblock upgrading serilog-settings-configuration.

**Pull request merged**